src/pymorize/cmorizer.py

```diff
@@ -145,6 +146,18 @@ def _post_init_data_request_variables(self):
         self._rules_expand_drvs()
         self._rules_depluralize_drvs()
 
+    def _post_init_read_dimensionless_unit_mappings(self):
```

Comment:
Could you add a docstring here @siligam?

Reply:
This is one of mine, I'll take care of it

Reply:
Done, see 1c5b69b. 🙂